static/js/components/email/lib.js

```diff
@@ -55,7 +56,13 @@ export const SEARCH_RESULT_EMAIL_CONFIG: EmailConfig = {
     emailState.inputs.body || '',
     emailState.searchkit.buildQuery().query,
     emailState.inputs.sendAutomaticEmails || false,
-  ])
+  ]),
+
+  extraUI: (emailCompositionType: string, setEmailCompositionType: (b: string) => void) => (
+    <EmailCompositionType
+      emailCompositionType={emailCompositionType}
+      setEmailCompositionType={setEmailCompositionType} />
+  )
```

Comment:
I think it would be better to avoid the function wrapper, as I said above - we'll need to add more props to this, and if they're all being passed in via this function wrapper it could get pretty messy.